tests/contracts/options.clar

```diff
@@ -0,0 +1,15 @@
+(define-public (some-truthy)
+    (ok (is-some (some 1)))
+)
+
+(define-public (some-falsy)
+    (ok (is-some (element-at? (list 1 2 3 4 5) u99)))
```

Comment:
Let's keep it simple for this unit test and use `none` directly.

Reply:
Done